draft-wang-ppm-differential-privacy.md

```diff
@@ -151,19 +159,96 @@ TODO Abstract
+is made possible by using a Verifiable Distributed Aggregation Function
+{{!VDAF=I-D.draft-irtf-cfrg-vdaf-06}}, the core cryptographic component of DAp.
+Execution of A VDAF involves: a large set of "Clients" who produce
+cryptographically protected measurements, called "reports"; a small number of
+"Aggregators" who consume reports and produce the cryptographically protected
+aggregate; and a "Collector" who consumes the aggregate result. Distributing
+the computation of the aggregate in this manner ensures that, as long as one
+Aggregator is honest, no attacker can learn an honest Client's measurement.
+
+Depending on the application, protecting the measurements may not be sufficient
+for privacy, since the aggregate itself can reveal privacy-sensitive
+information. As an illustrative example, consider using DAP/VDAF to summarize
+the distribution of the heights of respondents to a survey. If one of the
+respondents is especially short or tall, then their contribution is likely to
+skew the summary statistic in a way that reveals their height. Ideally, no
+individual measurement would have such a signficant impact on the aggregate
+result, but in general such leakage is inevitable.
+
+This intuition can be formalized by the notion differential privacy {{Dwo06}}.
+Differentially privacy is a property of an algorithm or protocol that computes
+some function of a set of measurements. We say the algorithm or protocol is
+"differentially private", or "DP", if the probability of observing a particular
+output does not change signficantly as a result of removing one of the
+measurements (or substituting it with another).
+
+VDAFs are not DP on their own, but they can be composed with a variety of
+mechanisms that endow them with this property. All such mechanisms work by
+introducing "noise" into the computation that is carefully calibrated for a
+number of application-specific parameters, including the structure and number
+of measurements and the desired aggregation function. [TODO: Anything else?]
+
+Noise can be introduced at various steps at the computation, and by various
+parties. Depending on the mechanism: the Clients might add noise to their own
+measurements; the Aggregators might add noise to their aggregate shares (the
+values they produce for the Collector); and the Collector might add noise to
+the aggregate result before publishing it. We refer to these classes of
+mechanisms as "Client-DP", "Aggregator-DP", and "Collector-DP" respectively.
+[TODO: I put "Collector-DP" here as a strawman. Would we actually want to use
+it?]
+
+In addition, it is possible to compuse different DP mechanisms. [TODO: Say why
+we would want to do this and give an illustrative example of combining
+Client-DP and Aggregator-DP.]
+
+[TODO: Transition text] This document defines a "DP policy" as the composition
+of an optional Client-DP mechanism and an optional Aggregator-DP mechanism
+with a VDAF.
```

Comment:
@junyechen1996 this is how I suggested we define "DP policy" in https://github.com/wangshan/draft-wang-ppm-differential-privacy/pull/5#pullrequestreview-1534034920.